Resources/views/Sync/comments.html.twig

```diff
@@ -0,0 +1,40 @@
+<div id="comments" class="js-hidden">
+    <div class="grid-row centered">
+        <div class="grid-content">
```

Comment:
these seem to be classes specific to your projects, so I don't think it should be there